### DISCUSSION TITLE
Have date fields different logic attached to them?

### DISCUSSION THREAD

**Discussion by MemoryFreezer** [firefly-iii/firefly-iii#9430] [Firefly III] [answered]
Hello all, I just have read an old post about some input to the date fields [Issue 4289](https://github.com/firefly-iii/firefly-iii/issues/4289). As the original author already stated, thanks a lot for the nice documentation and firefly-iii in general, its a nice project to tinker with.

Anyhow I recently tried to use the transaction booking and payment dates to figure out if I can recreate the following scenario:

I topped up my Revolut account with the debit card of my main banking account. This leads to the topped up amount being instantly available in Revolut but - lets say 5 days later - in my banking account. 

Is there anyway I can create a **transfer** that increases the destination account on a given date and decreases the source account on another date? Again this is not about deferred or delayed debit card / credit card payments and bookings.

Thanks in advance

**Comment by JC5** [comment, marked as answer]
Those dates are for the visualization only, they serve no other purpose. The source and destination share one date, so the amount always arrives instantly.

If you want, you can use a "buffer" asset account to simulate what happens IRL.
